pkg/actions/templates.go

```diff
@@ -24,12 +25,12 @@ import (
 // ListTemplates lists project templates of which Codewind is aware.
 // Filter them by providing flags
 func ListTemplates(c *cli.Context) {
-	templates, err := apiroutes.GetTemplates(
-		c.String("projectStyle"),
-		c.Bool("showEnabledOnly"),
-	)
-	if err != nil {
-		log.Printf("Error getting templates: %q", err)
+	projectStyle := c.String("projectStyle")
+	conID := strings.TrimSpace(strings.ToLower(c.String("conid")))
+	showEnabledOnly := c.Bool("showEnabledOnly")
+	templates, templatesErr := apiroutes.GetTemplates(projectStyle, conID, showEnabledOnly)
```

Comment:
Sometimes the connection id is the first parameter, sometimes it's in the middle and sometimes it's that last.
Since it's required by all the functions in apiroutes it would be good if it was always in the same place - probably as the first parameter since all api calls are made on a connection.

Reply:
Have changed so that conid is always the first parameter